stdlib/public/SwiftShims/Visibility.h

```diff
@@ -19,7 +19,11 @@
 #define SWIFT_STDLIB_SHIMS_VISIBILITY_H
 
 #if !defined(__has_feature)
-#define __has_feature(x) false
+#define __has_feature(x) 0
+#endif
+
+#if !defined(__has_attribute)
+#define __has_attribute(x) 0
```

Comment:
BTW, we have to define these here, as these are installed headers - we don't want to rely on external LLVM scripts here.